dhis2/dhis2-core
# chore: make embedded Tomcat HTTP port configurable

## Summary
This PR adds an environment variable called `DHIS2_HTTP_PORT`, which will be used to set the embedded Tomcat's HTTP port number.

### Manually test
1. Build the project
2. Go to `dhis-web-server/target`
3. Run: `export DHIS2_HTTP_PORT=8888`
4. Start the server by running: `./dhis.war`
5. Observe that the servers starts on port 8888
